vm.go

```diff
@@ -5591,6 +5592,10 @@ func (r *getPrivateRefId) exec(vm *vm) {
 }
 
 func (s valueStack) MemUsage(ctx *MemUsageContext) (memUsage uint64, err error) {
+	return ValuesMemUsage(s, ctx)
+}
+
+func ValuesMemUsage(s []Value, ctx *MemUsageContext) (memUsage uint64, err error) {
```

Comment:
do we need to export this function?

Reply:
Oh yeah goja is under the same package, yeah I'll make this a private function, nice catch!